app/build.gradle

```diff
@@ -142,132 +142,125 @@ androidComponents {
 dependencies {
 
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    def appcompat_version = '1.7.0'
-    implementation "androidx.appcompat:appcompat:$appcompat_version"
+
+    implementation libs.appcompat
```

Comment:
Minor nit: using parenthesis for most other deps here:
```suggestion
    implementation(libs.appcompat)
```

Reply:
eh actually further down it's more of a norm to not use them in this file - I like the idea of using them throughout just because it's forced if we move to kts vs Groovy, but nbd either way